src/main.cpp

```diff
@@ -26,6 +28,9 @@ namespace {
 constexpr int kFatalErrorOnStartupExitCode = 1;
 constexpr int kParseCmdlineArgsErrorExitCode = 2;
 
+constexpr char kScaleFactorEnvVar[] = "QT_SCALE_FACTOR";
+const QString kStrScaleFactor = QStringLiteral("ScaleFactor");
```

Comment:
Please don't introduce custom type prefixes. Instead name the constant according to its purpose, i.e. a config key as far as I understood.